theme/theme.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @file
+ * An Islandora module for downloading specified datastreams on objects.
+ */
+
+/**
+ * Implements hook_preprocess_theme().
+ */
+function islandora_cartographic_downloads_preprocess_islandora_cartographic_downloads(&$variables) {
```

Comment:
As this is the base implementation could probably just make this `template_preprocess`.